source/scss/blocks/intro.scss

```
@@ -167,3 +167,25 @@
     font-size: 48px;
   }
 }
+
+.text__word {
+  display: inline-block;
+  margin: 0 0 -0.5em 0;
+  overflow: hidden;
+  white-space: nowrap;
+  line-height: 1.2;
+}
+
+.text__word > span {
+    transform: translate3d(0, 1.25em, 0);
+    display: inline-block;
+    will-change: transform;
+}
+
+.intro__title.active .text__word span {
+  transform: translate3d(0, 0, 0);
+}
+
+.intro__date.active .text__word span {
+  transform: translate3d(0, 0, 0);
+}
```

Можно использовать перечисление, раз стили одни и те же:
```suggestion
.intro__title.active .text__word span,
.intro__date.active .text__word span {
  transform: translate3d(0, 0, 0);
}
```